controllers/fetchdata_controller.go

```diff
@@ -140,23 +143,23 @@ func NewCronJob(m *curatorv1alpha1.FetchData, scheme *runtime.Scheme) *batchv1.C
 										},
 										{
 											Name:  "DATABASE_NAME",
-											Value: m.Spec.DatabaseName,
+											Value: dbConfig.Database,
```

Comment:
Note during our call: let's just work towards using os.Getenv here and remove the dbConfig parameter from the function signature.